### PR TITLE
Added Webhook success message, corrected URL- closes #25, #24

### LINKED ISSUE #25: Add information on what to expect when checking the webhook.
#### Is your feature request related to a problem? Please describe.

Currently the README simply tells you to run a check to test the webhook but doesn't give guidance on what to expect.

#### Describe the solution you'd like
Add instructions based on the [content here](https://developers.facebook.com/docs/messenger-platform/getting-started/webhook-setup/). Here's what the relevant part states.

If your webhook verification is working as expected, you should see the following:
* `WEBHOOK_VERIFIED` logged to the command line where your node process is running.
* `CHALLENGE_ACCEPTED` logged to the command line where you sent the cURL request.

### PR DESCRIPTION
**Pull request by Zenix27**
#### Is your pull request related to a problem? Please describe
Closes #25 (issue) and also corrects the URL of #24 

#### Describe the solution you'd like
Added Webhook success message (after check) to give a clear understanding of what to expect after checking and also enhanced the doc by removing some typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,10 @@ docker-compose up -d
 ```bash
 curl -X GET "<YOUR HOST>/webhook?hub.verify_token=<YOUR VERIFY TOKEN>&hub.chalenge=CHALLENGE_ACCEPTED&hub.mode=subscribe&init_bot=true"
 ```
+If your webhook verification is working as expected, you should see the following:
+
+-```WEBHOOK_VERIFIED``` logged to the command line where your node process is running.\
```
Comment (elinguiuriel) [inline README.md:96]: please implement it in [messenger.py](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/9493a26b6ab7851bf2d49d59cb11af5cbe27d741/src/services/messenger.py)

Comment (Zenix27) [reply README.md:96]: I added instructions based on issue #25, as I understand the issue, the proposed changes are for the README part I guess.

**Comment by Zenix27** [comment]
URL to check the webhook was incorrect, changed ``hub.chalenge`` to ``hub.challenge``, which closes #24

File: README.md (hunk 1 above)
Comment (aboullaite) [reply README.md:96]: You're right @Zenix27! LGTM and it good to go. Thank you for taking time fixing those issues :)

Comment (Zenix27) [reply README.md:96]: Sure Thanks, @aboullaite @elinguiuriel

**Pull request merged**